### ISSUE TITLE
1080p 60fps videos play really laggy

### ISSUE THREAD

**Issue by Quiel245** [libre-tube/LibreTube#5027]
### Steps to reproduce

1. play any video in at least 1080p 60fps

### Expected behavior

That the video would play at least similarly to how it usually does on the normal youtube app, at 60fps

### Actual behavior

The video is running at really low frame rates when my device is capable of running it decently

### LibreTube version

0.19.0

### Android version

Android 13 TQ3A.230805.001

### Other details

I'm using lineageOs

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
We don't even have an option to select 60fps...

**Comment by Quiel245** [comment]
> We don't even have an option to select 60fps...

Oh, yeah, i meant like, if you choose the 1080p quality in a video that has 60fps support, but it's okay, i think i can watch everything in 720p...

**Comment by sp629** [comment]
i guess its upto device's hardware. like my device can't record 2160p 60fps so it doesn't play 2160p 60fps videos in any app like vlc mx lt yt. but my device support

**Comment by Quiel245** [comment]
Yeah it's that or maybe a rom bug, cuz like, it works just fine in the oficial YouTube app and other video players
